src/components/paragraphs/sum-calculator/sum-calculator.tsx

```diff
@@ -87,12 +87,19 @@ const SumCalculatorParagraph = ({costPerStudentTypes, appCost, progCosts, i20Cos
     >
       <div className="mx-auto flex max-w-7xl flex-col gap-20 pb-72">
         <div onFocus={onSelectFocus}>
+          <label
+            className="rs-mb-1 type-3 block"
+            id="sum-student-type"
+          >
+            Tell us who you are
+          </label>
           <SelectList
-            label="I am a/an _________ student"
+            ariaLabelledby="sum-student-type"
+            placeholder="Pick a level"
             options={[
-              {value: "undergraduate", label: "Undergraduate"},
-              {value: "highschool", label: "High School"},
-              {value: "graduate", label: "Graduate"},
+              {value: "undergraduate", label: "I am a/an Undergraduate student"},
+              {value: "highschool", label: "I am a/an High School student"},
+              {value: "graduate", label: "I am a/an Graduate student"},
```

Comment:
`a/an` is not necessary.
```suggestion
              {value: "undergraduate", label: "I am an Undergraduate student"},
              {value: "highschool", label: "I am a High School student"},
              {value: "graduate", label: "I am a Graduate student"},
```